### ISSUE TITLE
A way to set bookmarks for places on the map

### ISSUE THREAD

**Issue by rainbowdesign** [javisar/ONI-Modloader#42]
It would be cool to be able to add bookmarks for places like space, farm or oilwell to the map so you can quickly jump to that place without scrolling the whole map inbetween.

**Comment by fistak** [comment]
I don't think this is an issue, as you can zoom-out then in to any place in the world very quickly by using mod like `CameraControllerMod`.

**Comment by javisar** [comment]
You can use
Ctrl + [Number] to set the view to a place
Shift + [Number] to jump to that place

**Comment by rainbowdesign** [comment]
maybe request from klei to add screenshots to the shortcuts.

**Comment by rainbowdesign** [comment]
So you can see what keyshort points to where and can easily click on it. Ofcourse you can always keep a screenshot in your mind too... But well its just eyecandy maybe not that important
